src/index.html

```diff
@@ -125,24 +130,70 @@ <h1 class="text-white mb-6">
     <section>
       <div class="bg-background-off-white">
         <div class="max-width-container text-center">
-          <h4>How it Works</h4>
+          <h4 class="text-lightGray">How it Works</h4>
           <h1>Introducing the Raster Vision pipeline</h1>
-          <p class="small-body">
+          <p class="small-body text-lightGray">
             Example is showing a semantic segmentation task
           </p>
           <div class="sm:flex sm:flex-row gap-10 my-5">
-            <p class="medium-body my-10 sm:my-0">
-              ONE Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-              do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-            </p>
-            <p class="medium-body my-10 sm:my-0">
-              TWO Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-              do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-            </p>
-            <p class="medium-body my-10 sm:my-0">
-              THREE Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-              do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-            </p>
+            <div class="sm:w-1/3 mb-16">
+              <div class="sm:h-64 flex items-center justify-center">
+                <img src="images/rv_anim_ss_input.gif" class="m-auto sm:m-0" />
+              </div>
+              <div
+                class="sm:h-72 md:h-64 lg:h-44 flex flex-col justify-between mx-6 sm:mx-0"
+              >
+                <div>
+                  <h3><span class="text-teal">01</span> Data input</h3>
+                  <p class="medium-body my-10 sm:my-0">
+                    Input set of images and training data, optionally with Areas
+                    of Interest (AOIs) that describe where the images are
+                    labeled.
+                  </p>
+                </div>
+                <button class="secondary-btn">
+                  Start training with GroundWork
+                </button>
+              </div>
+            </div>
+            <div class="sm:w-1/3 mb-16">
+              <div class="sm:h-64 flex items-center justify-center">
+                <img src="images/img_rv-pipeline.png" class="m-auto sm:m-0" />
+              </div>
+              <div
+                class="sm:h-72 md:h-64 lg:h-44 flex flex-col justify-between mx-6 sm:mx-0"
+              >
+                <div>
+                  <h3>
+                    <span class="text-teal">02</span> Raster Vision pipeline
+                  </h3>
+                  <p class="medium-body my-10 sm:my-0">
+                    The computer vision pipeline configuration documentation is
+                    easy to read, reuse, and maintain.
+                  </p>
+                </div>
+                <a href="https://docs.rastervision.io/" target="_blank">
```

Comment:
When I added links to the hero buttons, I decided not to have them open in a new tab, mostly because I figured the point of a marketing site was to get users to the product and probably wasn't helpful to have open alongside it. I'm not entirely convinced that that's the best play, but for now I think that the buttons should all do the same thing. 

```suggestion
                <a href="https://docs.rastervision.io/">
```